includes/discovery/sensors/load/rfc1628.inc.php

```diff
@@ -5,7 +5,7 @@
 $load_data = snmpwalk_group($device, 'upsOutputPercentLoad', 'UPS-MIB');
 
 foreach ($load_data as $index => $data) {
-    $load_oid = ".1.3.6.1.2.1.33.1.4.4.1.5.$index";
+    $load_oid = ".1.3.6.1.2.1.33.1.4.4.1.5.".$index.".0";
```

Comment:
We've not had any complaints about these and I feel this might break things (doesn't look like we have any test data, I do though and none of it shows with a .0)

Reply:
Would it be something specific to the Galaxy 7000 ? I can (and will) provide test data for my units. But I did not find any other test data matching this RFC anyway so I cannot say if this change breaks anything or not ... 
I'll try to check the MIB itself to get convinced what is the "correct" way, and search for a proper way to handle this. In the meantime, I'll put a WIP on this.

Reply:
Hello,
I did change the behaviour to detect if the data comes as an array (meaning this ".0" is necessary) and do the necessary changes for the Galaxy to work without impacting existing device. 
I also added the test data from our device. 
This should now be a lot safer for any other device.